src/components/Nav/Nav.js

```diff
@@ -23,17 +31,23 @@ export default function Nav() {
         <S.Menu>
           {navList.map(({ id, link, text }) => (
             <S.ListStyle key={id}>
-              <S.AStyle href={link}>{text}</S.AStyle>
+              <S.AStyle onClick={() => goToUrl(navigate, link)}>
+                {text}
+              </S.AStyle>
             </S.ListStyle>
           ))}
         </S.Menu>
         <S.Aside>
           <S.Button>
             <S.SearchBar />
           </S.Button>
-          <Link to="/login">
-            <S.AsideButton>회원가입/로그인</S.AsideButton>
-          </Link>
+          {insertedToken ? (
+            <S.AsideButton onClick={() => logout()}> 로그아웃</S.AsideButton>

```

Comment:
```suggestion
            <S.AsideButton onClick={logout}> 로그아웃</S.AsideButton>
```
인자를 전달하지않는다면 함수 자체를 넣어줘도 됩니다

Reply:
넵 수정해서 다시 올리겠습니다!